### PR TITLE
CCD-3248 Monthly Review of All Security Flags - Comment out all suppr…

### PR DESCRIPTION
**Pull request by mikebrownccd**
https://tools.hmcts.net/jira/browse/CCD-3248
Monthly Review of All Security Flags

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: dependency-check-suppressions.xml [hunk 1]
```diff
@@ -1,56 +1,30 @@
...
 		<cve>CVE-2020-23171</cve>
-	</suppress>
-
-	<suppress>
-		<notes>We do not use: Spring Framework 5.0.5.RELEASE + Spring Security
-			(any version), see https://pivotal.io/security/cve-2018-1258
-			False positive confirmed.
-			Last control version: 5.5.1
-		</notes>
 		<cve>CVE-2018-1258</cve>
```
Comment (rajaram-nagarajan) [inline dependency-check-suppressions.xml:18]: shall we remove CVE-2018-1258 entirely?

Comment (mikebrownccd) [reply dependency-check-suppressions.xml:18]: suppression of this is required

Comment (rajaram-nagarajan) [reply dependency-check-suppressions.xml:18]: refer https://github.com/hmcts/ccd-case-disposer/pull/74/files

Comment (rajaram-nagarajan) [reply dependency-check-suppressions.xml:18]: agreed as invalid

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Review by dinesh1patel — APPROVED** [review]
LGTM

**Pull request merged**